streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBVersionedStore.java

```diff
@@ -641,6 +642,8 @@ private <T extends VersionedStoreSegment> PutStatus maybePutToLatestValueStore(
                     versionedStoreClient.deleteLatestValue(key);
                 }
                 return new PutStatus(true, foundTs);
```

Comment:
Seems `foundTs` should always be `SENTINEL_TIMESTAMP` here? Should we pass in `SENTINEL_TIMESTAMP` directly to make it more explicit?